### PR TITLE
Adds Vox Jumpsuits to Loadout Screen

### PR DESCRIPTION
**Pull request by PollardTheDragon**
## What Does This PR Do

Adds two otherwise-unused vox uniforms to the species-specific loadout tab.

Enables sensors for said vox uniforms.

Renames "Alien Clothes" to "Vox Jumpsuit" and "Alien Robes" to "Vox Robes"

## Why It's Good For The Game

Takes otherwise unused assets and adds them in a way that is fitting and does not impede gameplay. Adds additional drip, and diversifies the species loadout screen a bit.

## Images of changes

image
image
image
image

## Testing

Started many rounds, selected loadout gear, spawned with gear, set sensors to max.

## Changelog
:cl:
tweak: Renamed alien clothes to more fitting Vox Jumpsuit
tweak: Renamed alien robes to more fitting Vox Robes
tweak: Vox robes are now a suit slot item
tweak: Updated sprites
add: Vox robes now have a toggleable hood
add: Added vox clothes to loadout screen
tweak: Gave vox clothes suit sensors 
/:cl:

### CODE REVIEW

File: code/modules/client/preference/loadout/loadout_racial.dm [hunk 1]
```diff
@@ -58,3 +58,15 @@
 	display_name = "Cloth footwraps"
 	path = /obj/item/clothing/shoes/footwraps
 	slot = slot_shoes
+
+/datum/gear/racial/voxCasual
```
Comment (Burzah) [inline code/modules/client/preference/loadout/loadout_racial.dm:62]:
```suggestion
/datum/gear/racial/vox_casual
```
Small note. When naming procs or variables, you should use snake case.

File: code/modules/clothing/spacesuits/alien_suits.dm [hunk 2]
```diff
@@ -143,23 +143,22 @@
...
-	has_sensor = FALSE
 	icon = 'icons/obj/clothing/species/vox/uniforms.dmi'
 	species_restricted = list("Vox")
 	sprite_sheets = list(
 		"Vox" = 'icons/mob/clothing/species/vox/under/misc.dmi'
 		)
 
 /obj/item/clothing/under/vox/vox_casual
-	name = "alien clothing"
+	name = "Vox Jumpsuit"
```
Comment (Burzah) [inline code/modules/clothing/spacesuits/alien_suits.dm:153]:
```suggestion
	name = "Vox jumpsuit"
```
Jumpsuit should be lower case here so when examined the sentence will look proper.

**Review by Burzah — CHANGES_REQUESTED** [review]
Hello! Just a few small details here to look over and adjust.

**Comment by PollardTheDragon** [comment]
Adding updated sprites because the current ones are fugly and a decade old. On hold until then.

**Pull request closed without merging**